### ISSUE TITLE
Dependence on the `six` python package

### ISSUE THREAD

**Issue by eirikurj** [mdolab/pyoptsparse#61]
## Type of issue
What types of issue is it?

- Bugfix (non-breaking change which fixes an issue)

## Description

Seems that there is an explicit dependence on `six` which provides `iteritems`. 
Currently, there is a try/catch block which imports `iteritems` from `six`

### Steps to reproduce issue
On python 2 environment without having `six` installed running `python test_snopt_bugfix.py` in the `test` directory will generate:

```
Traceback (most recent call last):
  File "test_snopt_bugfix.py", line 75, in test_opt
    sol = opt(optProb, sens=sens)
  File "/home/mdolab/.local/lib/python2.7/site-packages/pyoptsparse/pySNOPT/pySNOPT.py", line 505, in __call__
    nS, ninf, sinf, ff, cu, iu, ru, cw, iw, rw)
  File "/home/mdolab/.local/lib/python2.7/site-packages/pyoptsparse/pySNOPT/pySNOPT.py", line 570, in _userfg_wrap
    gobj, gcon, fail2 = self._masterFunc(x, ['gobj', 'gcon'])
  File "/home/mdolab/.local/lib/python2.7/site-packages/pyoptsparse/pyOpt_optimizer.py", line 297, in _masterFunc
    result = self._masterFunc2(*args)
  File "/home/mdolab/.local/lib/python2.7/site-packages/pyoptsparse/pyOpt_optimizer.py", line 442, in _masterFunc2
    gobj = self.optProb.processObjectiveGradient(funcsSens)
  File "/home/mdolab/.local/lib/python2.7/site-packages/pyoptsparse/pyOpt_optimization.py", line 1361, in processObjectiveGradient
    for (objKey, dvGroup), _ in iteritems(funcsSens):
NameError: global name 'iteritems' is not defined
```

**Comment by johnjasa** [comment]
This is fully resolved by #62 and can be closed, is that correct?

**Comment by nwu63** [comment]
Whoops I totally forgot, yes that is correct. Closing now.
